addons/binding/org.openhab.binding.russound/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="russound"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <bridge-type id="rio">
+        <label>Russound RIO Device</label>
+        <description>Ethernet access point to Russound RIO control system (usually the main controller)</description>
+        
+        <channels>
+            <channel id="version" typeId="readonlystring"><label>Firmware Version</label><description>Firmware Version</description></channel>
+            <channel id="lang" typeId="lang"/>
+            <channel id="allon" typeId="button"><label>All Zones</label><description>Toggles All Zones</description></channel>
+        </channels>
+
+        <config-description>
+            <parameter name="ipAddress" type="text" required="true">
+                <context>network-address</context>
+                <label>IP or Host Name</label>
+                <description>The IP or host name of the Russound RIO access point</description>
+            </parameter>
+            <parameter name="ping" type="integer" required="false">
+                <label>Ping Interval</label>
+                <description>The ping interval in seconds to keep the connection alive</description>
+                <default>30</default>
+            </parameter>
+            <parameter name="retryPolling" type="integer" required="false">
+                <label>Retry Polling</label>
+                <description>The polling, in seconds, to retry a connection attempt</description>
+                <default>10</default>
+            </parameter>
+        </config-description>
+        
+    </bridge-type>
+
+    <bridge-type id="controller">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="rio"/>
+        </supported-bridge-type-refs>
+
+        <label>Russound Controller</label>
+        <description>Controller of Zones, Sources, etc</description>
+
+        <channels>
+            <channel id="type" typeId="readonlystring"><label>Model Type</label><description>Model Type</description></channel>
+            <channel id="ipaddress" typeId="readonlystring"><label>IP Address</label><description>IP Address</description></channel>

```

Comment:
Hm, is this any different from the IP adress of the bridge? Anyhow, as this is not changing dynamically, it should merely be added as a property, if at all.